### ISSUE TITLE
[A11Y] [iOS] Unable to linear navigate through items inside the 'VIEW TEMPLATE' popup.

### ISSUE THREAD

**Issue by spotukuchi** [ampproject/ampstart#407]
ENVIRONMENT:
Form factor : iPAD
Model : MC705LL/A
Version : 9.3.5(13G36)
Browser : Safari

PRE-REQUISITE:
Turn 'VoiceOver' ON. ( Settings->General->Accessibility->VoiceOver->ON )

STEPS TO REPRODUCE:
1. Navigate to the URL: https://ampstart.com/
2. Navigate to "Templates" link and double tap on it.
3. Navigate to 'VIEW TEMPLATE' in any of the sections and double tap on it.
4. Try to linear navigate through all the items in the Popup opened. 

ACTUAL RESULTS:
User is unable to navigate through any of the content in the Template view through linear navigation. Focus is moved onto the background text after double tapping on "VIEW TEMPLATE" button.

EXPECTED RESULTS:
User should be able to navigate through all the items in the popup opened after double tapping on 'VIEW TEMPLATE' button. Focus should stay within the popup opened, until the user closes it.
Background text should not be focussed.

ATTACHMENT:
Please see the video in the link below for reference.

https://drive.google.com/corp/drive/folders/0BwgKOOZP99EuUzVWT25TVzl2S00

**Comment by torch2424** [comment]
Unfortunately, this a bug with amphtml itself. It can be found at this issue: https://github.com/ampproject/amphtml/issues/4892

Once the changes come down, it should be good to go.

Closing...
